### ISSUE TITLE
foreach detected as "path with no return value" when if/else statement used

### ISSUE THREAD

**Issue by pielover928** [ItsDeltin/Overwatch-Script-To-Workshop#108]
I have this code:
```
foreach(...){}
if(...){
    return x;
} else {
    return y;
}
```

The compiler flags foreach as a place without a return value despite the loop breaking and following the flow of the code. By adding "return 0" at the bottom of the if/else block like so:

```
if(...){
return x;
}
else{
return y;
}
return 0;
```

the error goes away.

**Comment by ItsDeltin** [comment]
That's strange, it should be seeing the return in the if and else.

**Comment by ItsDeltin** [comment]
Fixed in [v1.2](https://github.com/ItsDeltin/Overwatch-Script-To-Workshop/releases/tag/v1.2).
